tests/reframe/config/system_properties.py

```diff
@@ -0,0 +1,2 @@
+
+ncorespernode=16
```

Comment:
how is this file used?

Reply:
it isn't anymore. It was a solution I had before ReFrame had support for describing processor properties in the config file... I took it out now